artifactory/1.11/using-artifactory.md

```diff
@@ -0,0 +1,130 @@
+## How to use Artifactory on DC/OS as a Docker Registry
+
+
+## Prerequisites
+
+- [DC/OS CLI installed](https://docs.mesosphere.com/1.11/cli/install/) and configured to use your cluster
+- A running installation of [Artifactory Pro](artifactory-pro.md) or
+  [Artifactory Enterprise](artifactory-enterprise.md) on DC/OS
+- [Artifactory-lb](artifactory-lb.md) installed with appropriate certificates
+  (or configured as an insecure registry, see below)
+- Ports 5000 to 5010 open on the public agent where Artifactory-lb is running.
+
+### (Optional) Configuring Artifactory as an insecure registry
+
+In case you'd like to run an insecure registry without using any TLS
+certificates, you can configure the nodes in your DC/OS cluster to work without
+certificates or security.
+
+This basically tells the Docker engine in each node to entirely disregard
+security for your registry. While this is relatively easy to configure the
+daemon in this way, it is very insecure. It does expose your registry to trivial
+MITM. Only use this solution for isolated testing or in a tightly controlled,
+air-gapped environment.
+
+Run this in all agent nodes of your cluster:
+
+```
+sudo cp /usr/lib/systemd/system/docker.service /etc/systemd/system/docker.service
+
+sudo vim /etc/systemd/system/docker.service
+
+add this line: Environment="DOCKER_OPTS=--insecure-registry artifactory-lb.marathon.mesos:5001”
+
+sudo systemctl daemon-reload
+
+sudo systemctl restart docker.service
```

Comment:
Restarting the docker service will shut down your containers, which DC/OS will respawn, but since we didn't pin anything to a node, they could come back anywhere, which could cause problems.

At the very least, I'd recommend a putting in warning about this. Maybe recommend doing this before you're running any services.

Reply:
We are requesting the setup to be done on all agent nodes.
I agree, that we can add put the warning or suggestion to do this setup before running any services.

Reply:
Can you add that to the README in this PR, or do you want to do a follow-up one?

Reply:
Ok. Will add to the main README.md ... will use this PR for the update.